com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/MultiInstanceHelpers.cs

```diff
@@ -15,16 +15,17 @@ namespace MLAPI.RuntimeTests
     /// </summary>
     public static class MultiInstanceHelpers
     {
-
         public static List<NetworkManager> NetworkManagerInstances = new List<NetworkManager>();
 
+        private static int s_OriginalTargetFrameRate = -1;
+
         /// <summary>
         /// Creates NetworkingManagers and configures them for use in a multi instance setting.
         /// </summary>
         /// <param name="clientCount">The amount of clients</param>
         /// <param name="server">The server NetworkManager</param>
         /// <param name="clients">The clients NetworkManagers</param>
-        public static bool Create(int clientCount, out NetworkManager server, out NetworkManager[] clients)
+        public static bool Create(int clientCount, out NetworkManager server, out NetworkManager[] clients, int targetFrameRate = 50)
```

Comment:
```suggestion
        public static bool Create(int clientCount, out NetworkManager server, out NetworkManager[] clients, int targetFrameRate = 60)
```

Reply:
I suggest using any other constant here then 50. Because 50 is also the standard tickrate for FixedUpdate. This will cause our tests to always run 1 FixedUpdate followed by 1 Update which would mean we might miss some edge cases.

Reply:
sure!